DesignSafe-CI/portal
# Bug/WC-122: Prevent a user from being added in multiple roles on a project.

## Overview: ##
Prevent the same user from being added in multiple roles on the same project, e.g. as a team member AND a co-PI.

## PR Status: ##


## Related Jira tickets: ##

* [WC-122](https://tacc-main.atlassian.net/browse/WC-122)

## Summary of Changes: ##

## Testing Steps: ##
1. In the project form, select a user as a co-PI and confirm that you cannot add the same user as a team member.
